### ISSUE TITLE
Aria label for chat tool codeblocks should be the title, not `code block n`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#249279]
Currently, using a screen reader, if I move focus to the chat tool response confirmation codelbock, I hear "Code block 1". I'd instead expect to hear "Run command in terminal: ls"

cc @jooyoungseo
